### PR TITLE
Brian-Jacob

### PR DESCRIPTION
**Pull request by PoundsXD**
#### Single-line Summary
**Today, Brian and Jacob paired together. It took about an hour and twenty minutes.**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on giving the process to the database
  2. Next, we started a new pg client
  3. Deleted the unecessary code.

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] I have entered the Pull Request title in the format of "firstname-lastname"
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

**Comment by PoundsXD** [comment]
We both programmed on one computer.

File: brian-jacob/server.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+// REVIEW: Install the middleware plugins so that our app is aware and can use the body-parser module
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static('./public'));
+
+
+// REVIEW: Routes for requesting HTML resources
+app.get('/new', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  // Put your response here... CRUD part in this is READ
```
Comment (luyounus) [inline brian-jacob/server.js:38]: Number:  ??
Method: ??
CRUD: Correct

File: brian-jacob/server.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  response.sendFile('new.html', {root: './public'});
+});
+
+
+// REVIEW: Routes for making API calls to use CRUD Operations on our database
+app.get('/articles', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
+  // 4, 5, Article.prototype.toHTML
```
Comment (luyounus) [inline brian-jacob/server.js:47]: Number:  Half-Correct
Method: Incorrect
CRUD: ??

File: brian-jacob/server.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+  })
+  .catch(function(err) {
+    console.error(err)
+  })
+});
+
+app.post('/articles', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
+  // 4, 5, Article.prototype.insertRecord
```
Comment (luyounus) [inline brian-jacob/server.js:61]: Number:  Half-correct
Method: Incorrect
CRUD: ??

File: brian-jacob/server.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+  })
+  .catch(function(err) {
+    console.error(err);
+  });
+});
+
+app.put('/articles/:id', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
+  // 2, 3, Article.prototype.insertRecord
```
Comment (luyounus) [inline brian-jacob/server.js:88]: Number:  Half-correct
Method: Incorrect
CRUD: ??

File: brian-jacob/server.js [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+  })
+  .catch(function(err) {
+    console.error(err);
+  });
+});
+
+app.delete('/articles/:id', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
+  // 2, 3, Article.truncateTable
```
Comment (luyounus) [inline brian-jacob/server.js:117]: Number:  Half-correct
Method: Incorrect
CRUD: ??

File: brian-jacob/server.js [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+  })
+  .catch(function(err) {
+    console.error(err);
+  });
+});
+
+app.delete('/articles', function(request, response) {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
+  // 2, 3, Article.truncateTable
```
Comment (luyounus) [inline brian-jacob/server.js:134]: Number:  Half-correct
Method: Correct
CRUD: ??

File: brian-jacob/server.js [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+app.listen(PORT, function() {
+  console.log(`Server started on port ${PORT}!`);
+});
+
+
+//////// ** DATABASE LOADER ** ////////
+////////////////////////////////////////
+function loadArticles() {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  // 2, 3, Article.prototype.insertRecord
```
Comment (luyounus) [inline brian-jacob/server.js:162]: Number:  Half-correct
Method: Incorrect
CRUD: ??

File: brian-jacob/server.js [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+          )
+        })
+      })
+    }
+  })
+}
+
+function loadDB() {
+  // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+  // 3, Article.fetchAll
```
Comment (luyounus) [inline brian-jacob/server.js:188]: Number:  Correct
Method: Incorrect
CRUD: ??

**Pull request merged**